tests/ssl-opt.sh

```diff
@@ -3014,8 +3014,13 @@ run_test    "Session resume using cache, DTLS: openssl server" \
 
 # Tests for Max Fragment Length extension
 
-if [ "$MAX_CONTENT_LEN" -lt "4096" ]; then
-    printf '%s defines MBEDTLS_SSL_MAX_CONTENT_LEN to be less than 4096. Fragment length tests will fail.\n' "${CONFIG_H}"
+if [ "$MAX_IN_LEN" -lt "4096" ]; then
+    printf '%s defines MBEDTLS_SSL_IN_CONTENT_LEN to be less than 4096. Fragment length tests will fail.\n' "${CONFIG_H}"
+    exit 1
```

Comment:
Note: use of `exit 1` here is wrong but it's a pre-existing problem. This PR is probably not the best place to fix it (the fix should be backported, while this PR shouldn't). I've created https://github.com/ARMmbed/mbedtls/issues/4471 to track it instead. (But if you feel like it, you're welcome to raise another PR to fix 4471 after this one.)

Reply:
Ack.